Sources/App/Views/PackageController/Builds/BuildIndex+Model.swift

```diff
@@ -89,20 +88,21 @@ extension BuildIndex.Model {
         var platform: App.Build.Platform
         var status: App.Build.Status
         var swiftVersion: App.SwiftVersion
-        var generatedDocs: Bool
+        var docStatus: DocUpload.Status?
 
         var isCompleted: Bool { status.isCompleted }
+        var isDocBuild: Bool {  docStatus != nil }
```

Comment:
I've fed through the actual doc status instead of coercing into a `Bool` early on. That way we can choose to make the icon carry status info as well if we so choose later on.

I've also renamed to `isDocBuild` instead of `generatedDocs`, because we can't actually be sure that docs have been generated.